app/views/content/steps-to-become-a-teacher/_step_2_understand_funding.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <p>You can apply for tuition fee and maintenance loans, even if you already have a student loan.</p>
 
-<p>If you're interested in teaching certain subjects, you might be able to get a tax-free bursary or scholarship. This money does not have to be paid back.</p>
+<p>If you're interested in teaching certain subjects, you might be able to get up to <%= v :scholarships_generic_maxshortened %> tax-free to support you while you're training. This money does not have to be paid back.</p>
```

Comment:
```suggestion
<p>If you're interested in teaching certain subjects, you might be able to get up a tax-free bursary or scholarship of up to <%= v :scholarships_generic_maxshortened %> to support you while you're training. This money does not have to be paid back.</p>
```
I know this was the wording in prod previously - any merit in making it a bit more explicit what we are talking about @emmaachesongray ?

Reply:
Agree, just need to remove 'up' before 'a tax-free' etc

Reply:
```suggestion
<p><p>If you're interested in teaching certain subjects, you might be able to get a tax-free bursary or scholarship of up to <%= v :scholarships_generic_maxshortened %> to support you while you're training. This money does not have to be paid back.</p></p>
```